Volume/Views/View Models/MorePublicationRow.swift

```diff
@@ -11,59 +11,36 @@ import SwiftUI
 /// `MorePublicationRow` displays the basis information about a publications the user is not currently following
 struct MorePublicationRow: View {
     @State private var didAddPublication = false
-    @State private var expanded = false
-    @State private var spacing: CGFloat = 12
     
     let publication: Publication
     
-    private func determineTruncation(_ geometry: GeometryProxy) {
```

Comment:
You can't make any of these changes in this file (except for the font change). 
Right now there are only 2 publications in `publicationsData`? since we're using a `LazyHStack` and you can't reuse publications in the list, but if you add more with varying description lengths, the spacing between rows is messed up unless the workaround here is included.